docs/sources/flow/get-started/install/kubernetes.md

```diff
@@ -20,20 +20,34 @@ weight: 200
 
 # Deploy {{% param "PRODUCT_NAME" %}} on Kubernetes
 
-{{< param "PRODUCT_NAME" >}} can be deployed on Kubernetes by using the Helm chart for {{< param "PRODUCT_ROOT_NAME" >}}.
+You can deploy {{< param "PRODUCT_NAME" >}} on Kubernetes using our Helm chart.
 
-## Before you begin
+## Dedicated guides
+
+We recommend using our dedicated guides for each telemetry signal to deploy a 
+production-ready {{< param "PRODUCT_NAME" >}} on Kubernetes:
+
+* For metrics, follow our dedicated [collect Prometheus metrics on Kubernetes][collect-prometheus] guide.
```

Comment:
nit: should we specific more specifically like `For Prometheus metrics,` and `For Loki logs,`. You are mentioning prometheus metrics later on in the sentence but given our stronger focus on otel these days we may want to be more explicit.

Reply:
You're right! will do :)